### ISSUE TITLE
Loss Curves

### ISSUE THREAD

**Issue by benliu961** [glinscott/nnue-pytorch#242]
Hello,

I am trying to produce the training and validation loss curves from when I run train.py. How would I get the curves?

Thank you!

**Comment by vondele** [comment]
https://pytext.readthedocs.io/en/master/visualize_your_model.html

**Comment by benliu961** [comment]
Do I need to train it in some other way to get the TensorBoard events? The website seems to have a tutorial that has a different way of training things and is also obsolete.

**Comment by vondele** [comment]
no, I don't think so, the file is generated somewhere in the tree during training.

**Comment by benliu961** [comment]
I cannot seem to find a /run/ folder. All I see is build, docs, env, lib, logs, scripts

**Comment by vondele** [comment]
I find such a file for example in `logs/lightning_logs/version_0/events.out.tfevents.1672585875` or `./run823/nnue-pytorch/exp/run_0_2022-03-31/default/version_0/events.out.tfevents.1648580018.nid05204.30411.0`  depending on the setup

On linux find it with `find . -name "*.tfevents.*"`

**Comment by benliu961** [comment]
Ah I found it. Thank you!
